src/main/java/cheese/task/Task.java

```diff
@@ -0,0 +1,72 @@
+package cheese.task;
+
+/**
+ * Represents a task with a description and complete/incomplete status.
+ */
+public class Task {
+    /** Description of task. */
+    private String description;
+
+    /** Whether task is complete or incomplete. */
+    private boolean isDone;
+
+    /**
+     * Constructs an instance of <code>Task</code>.
+     * 
+     * @param description Description of task.
+     */
+    protected Task(String description) {
+        this.description = description;
+        isDone = false;
+    }
+
+    /**
+     * Constructs an instance of <code>Task</code>.
+     * 
+     * @param isDone Whether task is complete or incomplete.
+     * @param description Description of task.
+     */
+    protected Task(boolean isDone, String description) {
+        this.isDone = isDone;
+        this.description = description;
+    }
+
+    public void setDone() {
+        isDone = true;
+    }
+
+    public void setNotDone() {
+        isDone = false;
+    }
+
+    /**
+     * Checks if task description contains given keyword, case insensitive.
+     * 
+     * @param keyword Keyword to search for.
+     * @return True, if description contains given keyword. False otherwise.
+     */
+    public boolean hasKeyword(String keyword) {
```

Comment:
I like that your boolean method sounds like a boolean.

Reply:
thank you ZhengLong for reviewing my code!